lib/middleware.js

```diff
@@ -65,7 +65,12 @@ module.exports = function wrapper(context) {
         }
 
         // server content
-        let content = context.fs.readFileSync(filename);
+        const defaultGenerator = ((context, filename, req, res) => context.fs.readFileSync(filename));
```

Comment:
Why passing `res, req`  as params ? (Use Cases)

Reply:
It is the signature of the function, the parameters that today are currently available to generate the content. 

It makes no difference today but, in the future, they may be required for the base middleware to perform some filtering, some operation (not foreseeable today)  and adding them back to the function would be a breaking change. Basically it is a case of the child function passing back what they received in the first place, to make it easier to chain.

Reply:
It is the signature of the function, the parameters that today are currently available to generate the content. 

It makes no difference today but, in the future, they may be required for the base middleware to perform some filtering, some operation (not foreseeable today)  and adding them back to the function would be a breaking change. Basically it is a case of the child function passing back what they received in the first place, to make it easier to chain.